.github/workflows/e2e-test.yaml

```diff
@@ -54,6 +54,7 @@ jobs:
         env:
           NO_MINIMIZE: true
           CI: true
+          REACT_APP_MIXPANEL_TOKEN: 6b8d3fb93ffd04f2e76325540c838624 # change mixpanel token to dev env
```

Comment:
This leaves two places for a single token, which is not a good practice considering maintenance. I think we can assign an empty token for it.